targets/audio

```diff
@@ -99,6 +99,13 @@ build_cras() {
             ln -sfT libasound.so.2 "$libasoundso"
         fi
         ALSALIBDIR="/usr/lib$archextrapath/alsa-lib"
+    elif release -le tumbleweed; then
+        install --minimal --asdeps alsa-devel
+        if [ $ARCH = "aarch64" ] || [ $ARCH = "x86_64"]; then
```

Comment:
yes, I realized this tonight as well, most probably this is the reason why X is not working in targets/xiwi